apps/portal/src/app/pay/testing-pay/page.mdx

```diff
@@ -16,7 +16,7 @@ Developers can turn on Test Mode to test both fiat-to-crypto transactions and cr
 
 ## Buy With Fiat
 
-By setting `testMode` to `true` for Buy With Fiat, you can enable test experiences for our underlying providers (Stripe, Kado, and Transak).
+By setting `testMode` to `true` for Buy With Fiat, you can enable test experiences for our underlying providers (Coinbase, Stripe, and Transak).
```

Comment:
This isn't true at the moment

Reply:
yeah, we'll have to fix that :D pretty important